cloudformation_templates/aws_kibana.json

```diff
@@ -0,0 +1,63 @@
+{
+  "AWSTemplateFormatVersion": "2010-09-09",
+  "Description": "Kibana AWS Elasticsearch domain with CloudWatch streaming",
+  "Parameters": {
+    "DomainName": {
+      "Type": "String",
+      "Description": "Domain name for AWS Elasticsearch service"
+    },
+    "UserIPs": {
+      "Type": "CommaDelimitedList",
+      "Description": "List of IP addresses that can access the Elasticsearch domain"
+    }
+  },
+
+  "Resources": {
+    "ElasticsearchDomain": {
+      "Type": "AWS::Elasticsearch::Domain",
+      "Properties": {
+        "DomainName": {"Ref": "DomainName"},
+        "EBSOptions": {
+          "EBSEnabled": true,
+          "VolumeSize": 15,
+          "VolumeType": "gp2"
```

Comment:
It's a $1/month difference for SSD instead of HDD. And I think `st1` has a 500Gb volume size minimum.